lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -564,3 +568,41 @@ func getRestrictedImeis(imeis []*lte_models.Imei) []*lte_mconfig.MME_ImeiConfig
 	}
 	return ret
 }
+
+func getNProbeConfig(networkID string) (*lte_mconfig.LIAgentD, *lte_mconfig.PipelineD_LiUes) {
+	liUes := &lte_mconfig.PipelineD_LiUes{}
+	liAgent := &lte_mconfig.LIAgentD{
+		LogLevel: protos.LogLevel_INFO,
+	}
+	ents, _, err := configurator.LoadAllEntitiesOfType(
+		networkID,
+		lte.NetworkProbeTaskEntityType,
+		configurator.EntityLoadCriteria{LoadConfig: true},
+		serdes.Entity,
+	)
+	if err != nil {
+		return liAgent, liUes
```

Comment:
Why having an error here doesn't matter?
The error is neither catch not passed

Reply:
I logged the error for now. 
Returning an error would break all mconfig builder unit tests since configurator.LoadAllEntitiesOfType() will always fail.
I will figure out a way to test this part of code in a separate PR.